com.unity.cinemachine/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.9.9] - 2023-12-13
+- Removed: API changes on CinemachineVirtualCameraBase.CancelDamping() and CinemachineBrain.TopCameraFromPriorityQueue().
```

Comment:
I wouldn't put this in the release notes, I'd just rename the 2.9.8 section to 2.9.9 and delete the bullets that correspond to the removed API (if any)

Reply:
What do we put in the changelog (We need at least one point)?

Reply:
We never release 2.9.8, so all the stuff in there will form the 2.9.9 section.  That's what I meant by "rename 2.9.8 to 2.9.9"